sopel/modules/reddit.py

```diff
@@ -209,3 +208,49 @@ def get_channel_sfw(bot, trigger):
         bot.say('%s is flagged as SFW' % channel)
     else:
         bot.say('%s is flagged as NSFW' % channel)
+
+
+@require_chanmsg('.setspoilfree is only permitted in channels')
+@commands('setspoilerfree', 'setspoilfree')
+@example('.setspoilfree true')
+@example('.setspoilfree false')
+def set_channel_spoiler_free(bot, trigger):
+    """
+    Sets the Spoiler-Free status (true or false) for the current channel.
+    Defaults to false.
+    """
+    if bot.channels[trigger.sender].privileges[trigger.nick] < OP:
+        return
+    else:
+        param = 'true'
+        if trigger.group(2) and trigger.group(3):
+            param = trigger.group(3).strip().lower()
+        spoiler_free = param == 'true'
+        bot.db.set_channel_value(trigger.sender, 'spoiler_free', sfw)
```

Comment:
Missed renaming one variable after pasting…

```suggestion
        bot.db.set_channel_value(trigger.sender, 'spoiler_free', spoiler_free)
```